.github/workflows/move-prover.yml

```diff
@@ -0,0 +1,16 @@
+name: Move Prover
+
+on: pull_request
```

Comment:
We should probably add something similar to https://github.com/MystenLabs/sui/blob/main/.github/actions/diffs/action.yml so that it only runs on PRs with move code.  Can be done later though.